py/uncategorized_behavior_report.py

```diff
@@ -0,0 +1,72 @@
+'''
+    Filename: uncategorized_behavior_report.py
+    Author: Brett Coker
+    Python Version: 3.6.1
+
+    Generates an uncategorized beahvior report for IBM - Network and saves it
```

Comment:
Whoops, you're totally right on this. I wrote this example for IBM and thought I scrubbed all of that out.